vcs/git.go

```diff
@@ -17,22 +21,144 @@ func MakeGit(rootPath string, basePath string) *Git {
 		log.Printf("Unable to chdir: %s", err.Error())
 	}
 
-	return &Git{basePath: basePath}
+	return &Git{
+		basePath: basePath,
+		logRX:    regexp.MustCompile("^((?:\\s*\\w{40}){2,})\\s+(.+)$"),
+	}
+}
+
+func (g *Git) GetCommitData(commit string) (*Commit, error) {
```

Comment:
Lets combine this function and `GetLastCommit`, seems they are doing same things inside, except first line
2 Options
1) Leave only 1 function, and pass additional args
2) Create 3rd function and move body there

Reply:
3. Leave it as is (also fine)